packages/loader/src/gltf/extensions/EXT_texture_webp.ts

```
@@ -35,9 +46,13 @@ class EXT_texture_webp extends GLTFExtensionParser {
   ): AssetPromise<Texture2D> {
     const webPIndex = schema.source;
     const { sampler, source: fallbackIndex = 0, name: textureName } = textureInfo;
+    let supportWebP = false;
+    if(this._supportWebP) {
```

ESLint

[error] Insert `·`

(prettier/prettier)